server/api/lib/mavat/planInstructions/parser/chartOneSix.js

```diff
@@ -0,0 +1,55 @@
+const { chartToArrayBuilder } = require('./chartToArrayBuilder');
+const { getFromArr } = require('./parsersUtils');
+const log = require('../../../log');
+
+// this function look for the correct columns for the given headers, and returns a factory embedded with these findings
+const rowAbstractFactoryChart1_6 = (firstPageOfTable, headersStartIndex) => {
+	// clean the headers
+	firstPageOfTable = firstPageOfTable.map(row => row.map(cell => cell.replace(/\n/g, ' ')
+		.replace(/ {2}/g, ' ')
+		.trim()));
+
+	if (headersStartIndex === -1) {
+		// it's an error
+		log.error('didn\'t find headers for some char 1.6');
```

Comment:
Passing a planId here requires me to pass planId into more software layers (at least 2 more) and change at least 25 lines in at least 15 different files. It will be a bit complicated.
The crawler that runs in production doesn't log the planId if it fails. Can you check the logs to see if we got a lot of "didn't find headers for *" to see if it worth investing in?